packages/divider-curve/src/assets/divider-curve-variation-one.tsx

```diff
@@ -0,0 +1,18 @@
+const DividerCurveOne = () => (
```

Comment:
There is no need to create 2 separate components here. You can simply have 2 React elements:

```tsx
export const divider1 = (
  <svg ...>
    ...
  </svg>
)

export const divider2 = (
  <svg ...>
    ...
  </svg>
)
```

Reply:
Updated to single element return based on `inverted` prop